ghcide/session-loader/Development/IDE/Session.hs

```diff
@@ -329,7 +333,9 @@ loadSessionWithOptions SessionLoadingOptions{..} dir = do
                let res = (map (renderCradleError ncfp) err, Nothing)
                modifyVar_ fileToFlags $ \var -> do
                  pure $ Map.insertWith HM.union hieYaml (HM.singleton ncfp (res, dep_info)) var
-               return (res, maybe [] pure hieYaml ++ concatMap cradleErrorDependencies err)
+               modifyVar_ filesMap $ \var -> do
+                 pure $ HM.insert ncfp hieYaml var
+               return (res,[])
```

Comment:
what's the reason to throw away the cradle error deps here?

Reply:
Accident, sorry!